packages/ramp-core/src/fixtures/legend/components/legend-options.vue

```diff
@@ -120,84 +120,90 @@
 
 <script lang="ts">
 import { GlobalEvents } from '@/api';
-import { LayerType } from '@/geo/api';
-import { Vue, Prop } from 'vue-property-decorator';
-import { Get, Sync, Call } from 'vuex-pathify';
+import { defineComponent, toRaw } from 'vue';
+import { call } from '@/store/pathify-helper';
 
 import { LegendStore } from '../store';
-import { LegendEntry, Controls } from '../store/legend-defs';
+import { Controls } from '../store/legend-defs';
 
-export default class LegendOptionsV extends Vue {
-    @Prop() legendItem!: LegendEntry;
-    @Call(LegendStore.removeLayerEntry) removeLayerEntry!: (entry: string) => void;
-
-    /**
-     * Display symbology stack for the layer.
-     */
-    toggleSymbology(): void {
-        if (this.legendItem._controlAvailable(Controls.Symbology)) {
-            this.legendItem.displaySymbology = !this.legendItem.displaySymbology;
-        }
-    }
+export default defineComponent({
+    name: 'LegendOptionsV',
+    props: ['legendItem'],
```

Comment:
let's keep the typing of the props if possible, i think this works: 
```
props: {
    legendItem: LegendEntry
}
```

Reply:
donethanks 👍